### PR TITLE
wrapping navigation bar icons into NavigationBarIconContainer

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This branch aims to add fixes to our `<DesktopNavigationBar />` and our `<MobileNavigationBar />`

- I created a `<NavigationBarIconContainer />` which role is to contain icons and have a fixed `width` and `height` to make text align on navigation bars
- I renamed `<SwitchIcon />` to `<WhiteSwitchIcon />`
- I renamed `<WorldIcon />` to `<WhiteWorldIcon />`
- I renamed `<HomeBackground />` to `<HomeMobileBackground />`
For more consitency
## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
There were no need to add or modify tests

## Screenshots
image
image

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx [hunk 1]
```diff
@@ -50,10 +52,10 @@ const ListItem = styled.div`
   padding: ${({ theme }) => theme.spaces.xs} 0 ${({ theme }) => theme.spaces.xs}
     ${({ theme }) => theme.spaces.xs};
   cursor: pointer;
+`;
 
-  div {
-    margin: 0 0 0 ${({ theme }) => theme.spaces.xs};
-  }
+const ListLabel = styled.div`
```
Comment (Fuzznimp) [inline src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx:57]: I think this should be a `p`.

File: src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx [hunk 2]
```diff
@@ -9,8 +9,8 @@ import { KeyIcon } from "../Icons/KeyIcon/KeyIcon";
 import { LockIcon } from "../Icons/LockIcon/LockIcon";
 import { NavBarCrossIcon } from "../Icons/NavBarCrossIcon/NavBarCrossIcon";
 import { RightChevron } from "../Icons/RightChevron/RightChevron";
-import { SwitchIcon } from "../Icons/SwitchIcon/SwitchIcon";
-import { WorldIcon } from "../Icons/WorldIcon/WorldIcon";
+import { WhiteSwitchIcon } from "../Icons/WhiteSwitchIcon/WhiteSwitchIcon";
+import { WhiteWorldIcon } from "../Icons/WhiteWorldIcon/WhiteWorldIcon";
```
Comment (Fuzznimp) [inline src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx:13]: What do you think about having `Icons/WorldIcon/(White|Black)WorldIcon`? The repo has already quite a lot of folder :).

File: src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx]: OK fixed 9944530 I did the same for `<MobileNavigationBar />`

File: src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx]: OK fixed ab546d4 I did the same for `SwitchIcon`

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**